src/components/TaskList.tsx

```diff
@@ -37,20 +53,26 @@ function ExpandableTaskItem({ item }: { item: TaskItem }) {
 
   return (
     <div key={item.id}>
-      <div
-        className="p-3 flex justify-between bg-gray-50 border-b items-center cursor-pointer"
-        onClick={handleClick}
-      >
-        {item.label}
-        {icon}
+      <div className="flex bg-gray-50 border-b center">
+        <div className="">
+          {" "}
```

Comment:
can we remove this?

Reply:
Idts, the div on the line 56 is containing the check input and the task label, so can be centralized. Any suggestion?